iothub/device/src/Messaging/OutgoingMessage.cs

```diff
@@ -73,16 +76,16 @@ public DateTime CreationTimeUtc
         /// <summary>
         /// The event creation time when sending data in a batch.
         /// </summary>
-        public DateTime CreationTimeBatchUtc
+        public DateTime BatchCreatedOnUtc
         {
-            get => GetSystemProperty<DateTime>(MessageSystemPropertyNames.CreationTimeUtc);
-            set => SystemProperties[MessageSystemPropertyNames.CreationTimeUtc] = value;
+            get => GetSystemProperty<DateTime>(MessageSystemPropertyNames.CreationTimeBatchUtc);
+            set => SystemProperties[MessageSystemPropertyNames.CreationTimeBatchUtc] = value;
         }
 
         /// <summary>
         /// The time when this message is considered expired
         /// </summary>
-        public DateTime ExpiryTimeUtc
+        public DateTime ExpiresOnUtc
```

Comment:
Align with naming pattern of other properties?

```suggestion
        public DateTime ExpiredOnUtc
```

Reply:
This field is in the future, unlike the other date times in here

Reply:
Expiration date is more likely in the future, so we opted for present/future tense.